src/app/find-help/view-offer-job/view-offer-job.component.ts

```diff
@@ -43,4 +45,49 @@ export class ViewOfferJobComponent implements OnInit {
       }
     );
   }
+
+  getResults() {
+    this.searchCriteria.industry = this.originalAccountQueryParams?.['industry'];
+    this.searchCriteria.mode = this.originalAccountQueryParams?.['mode'];
+    this.searchCriteria.contractType = this.originalAccountQueryParams?.['contractType'];
+    this.searchCriteria.workTime = this.originalAccountQueryParams?.['workTime'];
+    this.searchCriteria.language = this.originalAccountQueryParams?.['language'];
+    this.searchCriteria.location = this.originalAccountQueryParams?.['location'];
+
+    const PAGEREQUEST: Pageable = {
+      page: undefined,
+      size: 999999999,
+      sort: undefined,
+    };
+
+    this.jobsResourceService.listJob(PAGEREQUEST, this.searchCriteria).subscribe((results) => {
+      this.offerResults = results.content ?? [];
+      this.activeOffer = this.offerResults.find((x) => x.id === this.offerId);
+      this.activeIndex = this.offerResults.findIndex((x) => x.id === this.offerId);
+    });
+  }
+
+  slideOffer(index: number, direction: 'prev' | 'next') {
+    this.blurAnimate();
+    if (direction === 'prev') {
+      const SLIDE_PREV_DATA: JobOffer = this.offerResults[index - 1];
+      this.router.navigate([CorePath.Find, CategoryRoutingName.HEALTH, SLIDE_PREV_DATA.id]);
+      this.activeIndex = index >= 0 ? index - 1 : index;
+      this.offerId = this.activeIndex;
```

Comment:
Czy `offerId` to to samo co `activeIndex`?

Reply:
To wydaje mi się że to przypisanie jest błędne, ale może źle to czytam :)

Reply:
faktycznie coś jakby nie do końca spoko, zaraz popatrzę czy nie wkradł się babol, dzięki

Reply:
Musiałem przeoczyć, powinno być oczywiście tak - this.offerId = SLIDE_PREV_DATA.id;